jwst/assign_wcs/tests/test_wfss.py

```diff
@@ -235,21 +224,21 @@ def test_NIRISSBackwardDispersion():
          [15.66666438, 15.6661858 , 15.66570726, 15.66522876, 15.66475031, 15.66427189, 15.66379352]])
 
     # refactored call
-    # x, y = wcstools.grid_from_bounding_box(slit.meta.wcs.bounding_box)
-    # xdx, ydy, _, _ = model(x, y, wavelength, np.zeros(x.shape)+1)
+    x, y = grid_from_bounding_box(slit.meta.wcs.bounding_box)
+    xdx, ydy, _, _, _ = model(x, y, wavelength, np.zeros(x.shape)+1)
```

Comment:
Up above in line 150 you changed this call to use `order` in place of the hardcoded value "1". Is the same change warranted here?

Reply:
Yes, I'll change it. I'm running the tests for order=1 only.